modules/pel/peltool/peltool.py

```diff
@@ -327,6 +406,10 @@ def main():
 
         sys.exit(0)
 
+    if inBMC and args.list:
+        listOption("/var/lib/phosphor-logging/extensions/pels/logs/", config, args.extension, args.delete)
```

Comment:
Please use some constant, the same value (path) we hardcoded more than one places so it will be difficult if something (path) is changed in future.